google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -60,6 +60,7 @@ interfaces:
       - topic
     retry_codes_name: idempotent
     retry_params_name: default
+    timeout_millis: 30000
```

Comment:
The rpc timeout in the retry configuration is 60 seconds, this should probably be the same.